core/paramtable/base_table.go

```diff
@@ -52,6 +52,13 @@ const (
 
 	DefaultMinioBackupBucketName = "a-bucket"
 	DefaultMinioBackupRootPath   = "backup"
+
+	DefaultMilvusAddress              = "mlflow-stg"
+	DefaultMilvusPort                 = "19530"
+	DefaultMilvusAuthorizationEnabled = "yes"
```

Comment:
false

Reply:
Thank you @wayblink , i had fixed all that suggested by you.